src/hotspot/share/gc/g1/g1Allocator.hpp

```diff
@@ -198,6 +217,9 @@ class G1PLABAllocator : public CHeapObj<mtGC> {
                             bool* refill_failed,
                             uint node_index);
 
+  // Update the BOT for the last PLAB allocation.
+  inline void update_bot_for_allocation(G1HeapRegionAttr dest, size_t word_sz, uint node_index);
```

Comment:
```suggestion
  inline void update_bot_for_allocation(G1HeapRegionAttr dest, size_t word_sz, uint node_index);
```
```suggestion
  inline void update_bot_for_plab_allocation(G1HeapRegionAttr dest, size_t word_sz, uint node_index);
```

I would explicitly call this out as to be used for PLAB allocation. If changed, obviously needs updates to the callers as well.